### ISSUE TITLE
Retain Resized Panel Size in LocalStorage (between loads)

### ISSUE THREAD

**Issue by philcockfield** [kadirahq/react-storybook#466]
Would be great if the action-logger panel, and the index panel, didn't not lose their size between page loads.

![image](https://cloud.githubusercontent.com/assets/185555/18660028/64393536-7f62-11e6-9606-3cb6b967aa4d.png)

**Comment by arunoda** [comment]
Yeah. This is a pretty cool feature.
This is the place we are doing this: https://github.com/kadirahq/storybook-ui/
If you are looking to send a PR :)

**Comment by gnarf** [comment]
I would love a way to set the default height of that split differently as well

**Comment by gnarf** [comment]
I'd be happy to contribute to getting this feature, if you can point me at the bits of code that might help me set the initial height of these -- I tried looking for anything involving this split and got a bit lost

**Comment by UsulPro** [comment]
> to set the default height of that split differently as well

I agree, it is a good idea. I guess it could be included within the themes support implementation? 
https://github.com/kadirahq/react-storybook/issues/428#issue-175116679

**Comment by mnmtanish** [comment]
@gnarf that would be awesome! here's what you'll need:

**The split pane component**
https://github.com/kadirahq/react-split-pane

**Storybook UI code**
https://github.com/kadirahq/storybook-ui/blob/master/src/modules/ui/components/layout/index.js#L71,L111

**Comment by mnmtanish** [comment]
@UsulPro theme support might take a while. Most probably Storybook v3.
I've opened an issue on storybook-options addon repo. I'll try to add some more info there soon.

**Comment by UsulPro** [comment]
duplicates #869

**Comment by andrespch** [comment]
is there anyway to get the size of the pane within a story?

**Comment by Hypnosphi** [comment]
Do `window.innerHeight` and `window.innerWidth` work for you?

**Comment by andrespch** [comment]
I'll check that out @Hypnosphi , thanks for the help, I'm coming from the iOS world so even the most basic stuff is still a mystery :) I guess we can use `window` because the story in embedded in a `iFrame`? or is it not really related?

**Comment by GreatState** [comment]
I'm getting this on a deployed site. Any ideas why?
